engine/table/src/test/java/io/deephaven/engine/table/impl/remote/TestConstructSnapshot.java

```diff
@@ -26,6 +28,11 @@ public Boolean usePreviousValues(long beforeClockValue) {
             public boolean snapshotConsistent(final long currentClockValue, final boolean usingPreviousValues) {
                 return true;
             }
+
+            @Override
+            public UpdateGraph getUpdateGraph() {
+                return ExecutionContext.getContext().getUpdateGraph();
```

Comment:
FYI, we could provide a protected accessor to the UpdateGraph in RefreshingTableTestCase (which owns the test's execution context + test ugp).